mlflow/store/tracking/file_store.py

```diff
@@ -881,7 +880,6 @@ def _overwrite_run_info(self, run_info):
     def log_batch(self, run_id, metrics, params, tags):
         _validate_run_id(run_id)
         _validate_batch_log_data(metrics, params, tags)
-        _validate_batch_log_limits(metrics, params, tags)
```

Comment:
I think we should still keep this validation in the backend. Can we revert this change?